pkg/util/cluster/client.go

```diff
@@ -120,43 +109,48 @@ func PopulateAuthDetailsFromSecret(
 	secret *corev1.Secret,
 	useServiceAccount bool,
```

Comment:
```suggestion
	useCreatedServiceAccount bool,
```

Reply:
Both the initial service account token and certificate are for bootstrapping.

Note the flipping of the boolean value (i.e. `useServiceAccount == !isBootstrap`)

Reply:
The code has been updated.